src/convert.jl

```diff
@@ -295,3 +292,20 @@ function add_celldata!(vtk_celldata, mesh::TreeMesh, verbose)
 
   return vtk_celldata
 end
+
+
+function expand_filename_patterns(patterns)
+  filenames = String[]
+
+  for pattern in patterns
+    if startswith(pattern, '/') && !Sys.iswindows()
+      # Glob.glob does not support absolute paths; this workaround should enable this at least on
+      # non-Windows platforms
+      append!(filenames, glob(lstrip(pattern, '/'), "/"))
+    else
```

Comment:
Why do you not use `isabspath` as check and `relpath` to turn an absolute path into a relative path?

Reply:
Absolute paths are not forbidden, as far as I know, it's just that patterns may not start with a leading `/`. So in Windows it could be ok.

Since it boils down to the same spiel, I could do the following:
```suggestion
    if !Sys.iswindows() && isabspath(pattern)
      # Glob.glob does not support patterns with a leading `/` (= absolute paths on *nix operating systems)
      append!(filenames, glob(relpath(pattern)))
    else
```

Reply:
Well, your previous comment stated that
> Glob.glob does not support absolute paths

Thus, I thought this would be general. The basic question from my side is: Do we have problems with absolute path or with paths starting with `/`? The comment, condition, and solution should depend on this answer. If it's a problem with `/`, `Sys.iswindows()` or `isabspath(pattern)` should not play any role and we should use `startswith(pattern, '/')`. If it's a problem with absolute paths, `Sys.iswindows()` should also not play any role and we should use `isabspath(pattern)`.

Reply:
To be honest, I don't know the answer myself :shrug: From the implementation of the warning, I can only see that 
> Glob pattern cannot be empty or start with a / character

(see https://github.com/vtjnash/Glob.jl/blob/5caf444849574a7b618d920f8ce4f5ed60d71c8a/src/Glob.jl#L301)

Thus my thinking was: I am not sure what happens on Windows when specifying paths with a leading `/`. Therefore, I wanted to restrict my workaround to the case where I know what's going on, i.e., *nix-style OS, where it represents an absolute path). If not on Windows, we can also rely on the `relpath` function since we know that it will do the right thing.

But I really have no strong opinion here, I just wanted to quickly fix the issue of absolute paths on Linux/macOS not working at all.

Reply:
The error message seems to indicate that it doesn't work with paths starting with `/`. Thus, I would propose to use this as check and `relpath` to get rid of it. Windows also supports `/` as path separator, so it should behave well. If not, we can always fix it when users report problems.

Reply:
Implemented in d2d15c7.